minkindr/include/kindr/minimal/implementation/rotation-quaternion-inl.h

```diff
@@ -472,13 +496,24 @@ RotationQuaternionTemplate<Scalar>::log() const {
 }
 
 template<typename Scalar>
-bool RotationQuaternionTemplate<Scalar>::isValidRotationMatrix(const RotationMatrix& matrix) {
-  const Scalar kThreshold = static_cast<Scalar>(1.0e-7);
-  if (std::fabs(matrix.determinant() - static_cast<Scalar>(1.0)) > kThreshold) {
+bool RotationQuaternionTemplate<Scalar>::isValidRotationMatrix(
+    const RotationMatrix& matrix) {
+  constexpr Scalar kThreshold = static_cast<Scalar>(1.0e-8);
```

Comment:
can we restore the old threshold of 1.0e-7 again, to avoid problems with existing code?